### DISCUSSION TITLE
这套配置必须要用虚拟机吗？

### DISCUSSION THREAD

**Discussion by mouyase** [kkkgo/PaoPaoGateWay#91] [问和答]
本人家里目前的设备是两台软路由，一台拨号一台出国，用的是旁路网关的方式，经人推荐了解到这个项目，但是看了看文档似乎只适合用虚拟机来跑？
看起来理论上应该是支持多设备的，不知道有没有大佬有过一些经验分享呢？

**Comment by kkkgo** [comment]
那你iso写u盘跑实体机也不是不行。

**Reply by mouyase** [reply]
疑惑于带宽问题，虚拟机上似乎内网不会有带宽的损耗，如果是两台物理机部署的情况下，千兆网络的情况下，是不是会带来带宽损耗呢

**Reply by kkkgo** [reply]
你的代理速度大于1000Mbps再考虑损耗问题。

**Reply by mouyase** [reply]
可能是我理解有些问题。因为之前我是做旁路由的配置，非代理也会有带宽损耗问题，最高只能跑到600-700左右，跑不到945Mbps。
不过似乎看起来如果是使用ppgw+ppdns，可以做到仅海外走网关？

**Reply by kkkgo** [reply]
是的，如果你按照文档推荐的拓扑，仅有需要代理的流量会经过网关。

**Reply by mouyase** [reply]
感谢解答😀，看来新的学习之旅就要开始了

**Reply by mouyase** [reply]
试了试，似乎用几种方式写入U盘都不能正常引导。

使用Rufus写入U盘会提示不支持的镜像。
使用UltraISO写入后启动系统都会卡在系统引导。
使用balenaEtcher写入后系统会直接跳过从U盘中启动。

也许需要给镜像添加类似GRUB的东西？

**Reply by mouyase** [reply]
看了一下，iso里是有grub的，但是为啥写入到磁盘里面他就不启动呢，不太明白

我在试着用grub4dos做引导

**Comment by kkkgo** [comment]
PaoPaoGateway现在已支持Arm64版本镜像，参考： https://github.com/kkkgo/PaoPaoGateWay/discussions/214
